components/CommentModal/CommentModal.js

```diff
@@ -0,0 +1,124 @@
+import React, {useState} from "react";
+import { Button, Modal } from 'react-bootstrap'
+import { useAuth } from "../../components/auth"
+import { useProfile, useMember } from "../db"
+import { useEditTestimony } from "../db/testimony/useEditTestimony"
+import * as links from "../../components/links"
+
+const GetMemberEmail = (Id) => {
+  const { member } = useMember(Id)
+  return member ? member.EmailAddress : null
+}
+
+const CommentModal = (props) => {
+  const useTestimonyTemplate = true
+  const testimonyTemplate = 
+`Why I am qualified to provide testimony:
+
+Why this bill is important to me:
+  
+My thoughts:
+  `
+  const defaultTestimony = useTestimonyTemplate ? testimonyTemplate : "My comments on this bill..."
+  const [testimony, setTestimony] = useState(props.testimony ? props.testimony : {content: defaultTestimony})
+  
+    const bill=props.bill
+    const showTestimony=props.showTestimony
+    const handleCloseTestimony=props.handleCloseTestimony
+
+    const { user, authenticated } = useAuth()
+    const { profile } = useProfile()
+    const representativeId = profile && profile.representative ? profile.representative.id : null
+    const senatorId = profile && profile.senator ? profile.senator.id : null
+    const senatorEmail = senatorId ? GetMemberEmail(senatorId) : ""
+    const representativeEmail = representativeId ? GetMemberEmail(representativeId) : ""
+    const edit = useEditTestimony(user.uid, bill.BillNumber)
+    const positionMessage = "Select my support..(required)"
+    
+    const url = `mailto:${senatorEmail},${representativeEmail}?subject=My testimony on Bill ${bill ? bill.BillNumber : ""}&body=${testimony ? testimony.content : ""}`
+
+    const defaultPosition = testimony && testimony.position ? testimony.position : undefined
+  
+    const defaultAnonymous = testimony && testimony.anonymous ? testimony.anonymous : false
+    const defaultContent = testimony && testimony.content ? testimony.content : defaultTestimony
+    
+    const publishTestimony = async () => {
+      if (testimony.position == undefined || testimony.position == positionMessage) { return }
```

Comment:
Consider adding a validation error message to the form in this case so the user can more easily tell why the form didn't submit

Reply:
Good idea. I changed the publish button to be "Select your support to publish" if they haven't chosen a position to explain why it doesn't submit.
![image](https://user-images.githubusercontent.com/67333843/157488088-9248b82b-6dbd-4566-bec7-8fe529399515.png)